myapp/src/components/Priority.js

```diff
@@ -51,11 +51,8 @@ IconContainer.propTypes = {
   value: PropTypes.number.isRequired,
 };
 
-export default function CustomizedRatings(props) {
+const CustomizedRatings = ({ name, value, label, onChange }) => 
```

Comment:
customIcons export to utils and change to array of objects

Reply:
I deleted them because they were never in use.